android/fastlane/Fastfile

```diff
@@ -45,4 +57,30 @@ platform :android do
         )
     end
 
+    desc "Upload Android App to Google Play"
+    lane :upload_to_playstore do |options|
+        ensure_env_vars(
+            env_vars: ["GOOGLE_SERVICE_ACCOUNT_JSON"]
+        )
+
+        version_code = options[:version_code]
+        version_name = options[:version_name]
+        production_delivery = options[:production_delivery]
+
+        if [version_name, version_code, production_delivery].include?(nil)
+            raise "'nil' passed as parameter! Aborting..."
+        end
+
+        track = production_delivery === true ? "production" : "beta" # ToDo LUN-29
+
+        upload_to_play_store(
+            version_code: version_code,
+            version_name: version_name,
+            track: "internal", # ToDo LUN-29
+            release_status: "draft", # ToDo LUN-29
+            package_name: "com.lunesappui",
```

Comment:
Is this package name already decided? I don't like it to be honest and I think it should rather be `app.lunes`.